### DISCUSSION TITLE
Have a button to toggle focus on chat while in a multiplayer game.

### DISCUSSION THREAD

**Discussion by 19peytonsmith** [ppy/osu#25840] [Ideas] [answered]
Motivation: Sometimes during breaks while playing a beatmap in multiplayer I want to chat, but I end up having to grab my mouse to click on the small chat box (the size of the chatbox is another issue entirely), and then I can begin typing, which is cumbersome. It would be nice to instead be able to press a button, or during breaks have it auto-focused so that I can begin typing within manually doing so?

**Comment by bdach** [comment, marked as answer]
This already exists. Currently the hotkey is <kbd>Enter</kbd> but it is rebindable.

**Reply by 19peytonsmith** [reply]
Oh that makes sense, thank you! I tried typing Space & 'T' but neither of those worked so I thought it didn't exist, should've tried more 😅
